### PR TITLE
Allow setting the overlay color and padding programmatically

### PR DESCRIPTION
**Pull request by eygraber**

### CODE REVIEW

File: scissors/src/main/java/com/lyft/android/scissors/CropView.java [hunk 1]
```diff
@@ -113,6 +113,25 @@ protected void onSizeChanged(int w, int h, int oldw, int oldh) {
     }
 
     /**
+     * Sets the color of the viewport overlay
+     *
+     * @param color The color to use for the viewport overlay
+     */
+    public void setViewportOverlayColor(@ColorInt int color) {
+        viewportPaint.setColor(color);
```
Comment (eveliotc) [inline scissors/src/main/java/com/lyft/android/scissors/CropView.java]: Can you please use this method from `initCropView` and update config as well?

File: scissors/src/main/java/com/lyft/android/scissors/CropView.java [hunk 2]
```diff
@@ -113,6 +113,25 @@ protected void onSizeChanged(int w, int h, int oldw, int oldh) {
...
+    }
+
+  /**
+   * Sets the padding for the viewport overlay
+   * @param padding The new padding of the viewport overlay
+   */
+  public void setViewportOverlayPadding(int padding) {
+        touchManager.setViewportOverlayPadding(padding);
```
Comment (eveliotc) [inline scissors/src/main/java/com/lyft/android/scissors/CropView.java]: Please update config instead.

**Comment by eveliotc** [comment]
Thanks for this, can you please make that you have signed CLA as per our contribution guidelines?

https://github.com/lyft/scissors/blob/master/CONTRIBUTING.md

**Comment by eygraber** [comment]
@eveliotc CLA has been signed

File: scissors/src/main/java/com/lyft/android/scissors/CropView.java (hunk 1 above)
Comment (eygraber) [reply scissors/src/main/java/com/lyft/android/scissors/CropView.java]: Done

File: scissors/src/main/java/com/lyft/android/scissors/CropView.java (hunk 2 above)
Comment (eygraber) [reply scissors/src/main/java/com/lyft/android/scissors/CropView.java]: Done

**Comment by eygraber** [comment]
Sorry about the format. Not sure how that happened.

**Pull request merged**